support/cas-server-support-oidc-core-api/src/main/java/org/apereo/cas/oidc/discovery/OidcServerDiscoverySettings.java

```diff
@@ -119,4 +119,24 @@ public String getIntrospectionEndpoint() {
     public String getRevocationEndpoint() {
         return this.serverPrefix.concat('/' + OidcConstants.BASE_OIDC_URL + '/' + OidcConstants.REVOCATION_URL);
     }
+
+    @JsonProperty("backchannel_logout_supported")
+    public boolean getBackchannelLogoutSupported() {
+        return true;
+    }
+
+    @JsonProperty("backchannel_logout_session_supported")
+    public boolean getBackchannelLogoutSessionSupported() {
+        return true;
+    }
+
+    @JsonProperty("frontchannel_logout_supported")
+    public boolean getFrontchannelLogoutSupported() {
+        return true;
+    }
+
+    @JsonProperty("frontchannel_logout_session_supported")
+    public boolean getFrontchannelLogoutSessionSupported() {
+        return true;
+    }
```

Comment:
All such settings should be controllable via CAS settings, in scenarios where a deployment would want to forcefully turn off logout.
Functionality related to these settings should also turn itself on/off based on the value.

Reply:
OK. I will do that. By default, I'll kept logout enabled.